src/web/modules/gateway_accounts/gatewayAccount.model.ts

```diff
@@ -110,7 +110,7 @@ class GatewayAccount extends Validated {
       type: this.isLive() ? 'live' : 'test',
       service_name: this.serviceName,
       analytics_id: this.analyticsId,
-      requires_3ds: this.provider === 'stripe' ? 'true' : 'false'
+      requires_3ds: this.isLive() ? (this.provider !== 'stripe' ? 'true' : 'false'): 'false'
```

Comment:
2 ternary operators makes this logic hard to follow, maybe being more verbose might help readability? 

Something like 
```js
if (this.isLive() && this.provider !== 'stripe') {
  payload.required_3ds = 'true'
} else { 
  payload.required_3ds = 'false'
}
```

Reply:
👍 Updated